### ISSUE TITLE
Idention Warning

### ISSUE THREAD

**Issue by ohm-meter** [martin-helmich/typo3-typoscript-lint#109]
Hi,
I have that code:
```
[request.getPageArguments().get('tx_news_pi1')['news'] > 0]
    page.meta.og:url >
    page.meta.og:title >
    page.meta.og:site_name >
    page.meta.og:description >
    page.meta.og:image >
[global]
```

and i got this warning:
```
CHECKSTYLE REPORT
=> /public/typo3conf/ext/theme/Configuration/TypoScript/Page/meta.typoscript.

  75 Expected indent of 0 spaces.
  76 Expected indent of 0 spaces.
  77 Expected indent of 0 spaces.
  78 Expected indent of 0 spaces.
  79 Expected indent of 0 spaces.

```

What can i do ?

**Comment by martin-helmich** [comment]
This is configurable. Have a look at [the README](https://github.com/martin-helmich/typo3-typoscript-lint#indentation) for more information:

> By default, the indentation sniff expects code inside TypoScript conditions to be **not** indented. You can change this behaviour by setting the `indentConditions` flag for the indentation sniff to `true` in your `typoscript-lint.yml` configuration file (see below).
